test/gtest/uct/ib/test_ib.cc

```diff
@@ -180,6 +184,19 @@ UCS_TEST_P(test_uct_ib_addr, address_pack) {
     test_address_pack(0xdeadfeedbeefa880ul);
 }
 
+UCS_TEST_P(test_uct_ib_addr, address_pack_path_mtu, "IB_PATH_MTU=2048") {
+    uct_ib_iface_t *iface = ucs_derived_of(m_entities.front()->iface(),
+                                           uct_ib_iface_t);
+    size_t addr_len = uct_ib_iface_address_size(iface);
+    std::vector<char> buffer(addr_len);
+    uct_ib_address_t *addr = (uct_ib_address_t*)&buffer[0];
+    uct_ib_iface_address_pack(iface, addr);
+    uct_ib_address_pack_params_t params;
+    uct_ib_address_unpack(addr, &params);
+    EXPECT_TRUE((params.flags & UCT_IB_ADDRESS_PACK_FLAG_PATH_MTU) != 0);
```

Comment:
no need for `!=0`

Reply:
as I remember on newer compilers such evaluation provided compilation warning...
ok, fixed

Reply:
fix is missing

Reply:
yep, missed